src/main/java/com/facebook/openwifirrm/DeviceConfig.java

```diff
@@ -79,15 +79,6 @@ public static DeviceConfig createDefault() {
 		config.enableRRM = true;
 		config.enableConfig = true;
 		config.enableWifiScan = true;
-		config.boundary = null;
-		config.location = null;
-		config.allowedChannels = null;
-		config.allowedChannelWidths = null;
-		config.autoChannels = null;
-		config.userChannels = null;
-		config.allowedTxPowers = null;
-		config.autoTxPowers = null;
-		config.userTxPowers = null;
```

Comment:
We need to keep them as the optimization algorithms use these default values as the conditions to throw an exception or trigger some certain logic. :)

Reply:
Java automatically initializes uninitialized Object member variables to null, so having these does not add literally - but Jeffrey told me to keep these here anyways just for extra clarity for developers.